src/System.Runtime.Extensions/src/System/Globalization/Extensions.cs

```diff
@@ -0,0 +1,96 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
Can we just add this to API compat baseline instead of adding the whole implementation back?

Reply:
This failure is in GenFacades, not APICompat. So I believe not?

Reply:
`<GenFacadesIgnoreMissingTypes Condition="'$(TargetGroup)'=='uapaot'">true</GenFacadesIgnoreMissingTypes>`

Used in number of places already.

Reply:
Perfect - thanks!